src/sql/azdata.proposed.d.ts

```diff
@@ -432,18 +432,31 @@ declare module 'azdata' {
 		azurePortalEndpoint?: string;
 	}
 
-	export interface PromptFailedResult {
+	export interface PromptFailedResult extends ProviderError {
 		/**
-		 * Error code used for non-user cancelled sign in errors
+		 * Type guard for differentiating user cancelled sign in errors from other errors
+		 */
+		canceled: boolean;
```

Comment:
Remove this, it's already defined in azdata.d.ts. This here is just adding in the additional ProviderError properties.